components/pedigree/resources/src/main/resources/pedigree/editor.css

```diff
@@ -606,6 +606,28 @@
   width: 48%;
   float: right;
 }
+.menu-box .field-deceasedAge {
+  width: 47%;
+  float: left;
+}
+.menu-box .field-deceasedCause {
+  width: 48%;
```

Comment:
Okay, so then the whole layout needs to be fixed (but not now). There's no good reason to give different widths to blocks that are obviously meant to each occupy half.